### PR TITLE
Feat/edit delete

### PR DESCRIPTION
**Pull request by SHIKHASINGH1506**
1. User can edit his own posts by clicking on more icon on the post
2. S(He) can delete from the same more icon

### CODE REVIEW

**Review by sashaboi — APPROVED** [review]
Works perfectly as expected.

A suggestion for future versions : the edit button could be on the top right of the post card than on the bottom as is on a lot of current social media sites

File: src/features/post/components/post-form/index.jsx [hunk 1]
```diff
@@ -45,20 +68,20 @@ const PostForm = ({modalVisibilityHandler}) => {
...
             onChange={setPostFieldHandler}
             placeholder="What's happening with you ?" ></textarea>
           <div className='flex justify-between'>
-            <div className="icons flex">
+            {/* <div className="icons flex">
               <button className='p-2 mx-2'><i className="fa-solid fa-image"></i></button>
               <button className='p-2 mx-2'><i className="fa-solid fa-face-smile"></i></button>
-            </div>
-            <button 
+            </div> */}
```
Comment (JowelTisso) [inline src/features/post/components/post-form/index.jsx:81]: Remember to remove unused code

File: src/features/auth/authSlice.js [hunk 2]
```diff
@@ -15,7 +15,7 @@ export const loginUser = createAsyncThunk(
     try{
       const {data} = await login(loginCreds);
       localStorage.setItem('token', data.encodedToken);
-      localStorage.setItem('user', JSON.stringify(data.user));
+      localStorage.setItem('user', data.foundUser.username);
       localStorage.setItem('isLoggedIn', true);
       return data;
```
Comment (JowelTisso) [inline src/features/auth/authSlice.js:20]: You can abstract out the local storage function for reusabilty

**Review by JowelTisso — APPROVED** [review]
* Clean and modular code
* Remember to remove console logs
* good atomic CSS approach
* Rest looks good

**Pull request merged**